### PR TITLE
Instrumentation to answer #5765

### PR DESCRIPTION
**Pull request by jwhonce**
* currently wired to localhost:8888 to prevent access from off machine

Signed-off-by: Jhon Honce <jhonce@redhat.com>

### CODE REVIEW

File: pkg/api/server/server.go [hunk 1]
```diff
@@ -145,6 +147,20 @@ func (s *APIServer) Serve() error {
...
+			goRuntime.SetMutexProfileFraction(1)
+			goRuntime.SetBlockProfileRate(1)
+			s.pprof = &http.Server{Addr: "localhost:8888", Handler: pprofMux}
+			err := s.pprof.ListenAndServe()
+			if err != nil && err != http.ErrServerClosed {
+				logrus.Warn("Profiler Service failed: " + err.Error())
+			}
+		}()
+	}
+
```
Comment (TomSweeneyRedHat) [inline pkg/api/server/server.go:163]: Neat, I didn't realize logrus let you setup a function like this.

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests

**Comment by rhatdan** [comment]
This needs a rebase to pass the tests.

**Pull request merged**